api/nodes/views.py

```diff
@@ -1652,7 +1653,14 @@ def get_queryset(self):
 
     # overrides ListCreateAPIView
     def perform_create(self, serializer):
-        serializer.save(node=self.get_node())
+        user = get_user_auth(self.request).user
+        node = self.get_node()
+        try:
+            fork = serializer.save(node=node)
+            mails.send_mail(user.email, mails.FORK_COMPLETED, title=fork.title, guid=fork._id, mimetype='html')
```

Comment:
We could try making this something that happens with a query parameter, but that would essentially be special casing it, more than it is already. 

It's uncomfortable implementing this type of notification without a more generalized system which doesn't just apply to only forks, but that's not the scope of the ticket.